### DISCUSSION TITLE
Does ZED have a feature equivalent to VSCode?

### DISCUSSION THREAD

**Discussion by odevnicolas** [zed-industries/zed#6672] [Feature Requests]
<img width="347" alt="Screenshot 2024-01-02 at 11 03 15" src="https://github.com/zed-industries/community/assets/125375345/25e50714-ec78-4067-a114-242bc608d533">

 I wanted to search for a specific word in all files at once, what I found was just the default cmd + f that searches for words only in the current file

**Comment by JosephTLyons** [comment]
Hey @odevnicolas, try `cmd-shift-f` to open up a `project search` multi-buffer. What's neat about these in Zed is that they are live views into the underlying buffers, which means you can edit the contents of the search results and when you save, it writes the changes out to each file.
